---
title: SanitizationException
second_title: Aspose.PDF для справочника API .NET
description: Инициализирует новый экземплярSanitizationExceptionaspose.pdf.sanitization/sanitizationexception класс.
type: docs
weight: 10
url: /ru/net/aspose.pdf.sanitization/sanitizationexception/sanitizationexception/
---
## SanitizationException() {#constructor}

Инициализирует новый экземпляр[`SanitizationException`](../../sanitizationexception) класс.

```csharp
public SanitizationException()
```

### Смотрите также

* class [SanitizationException](../../sanitizationexception)
* пространство имен [Aspose.Pdf.Sanitization](../../sanitizationexception)
* сборка [Aspose.PDF](../../../)

---

## SanitizationException(string) {#constructor_2}

Инициализирует новый экземпляр[`SanitizationException`](../../sanitizationexception) класс.

```csharp
public SanitizationException(string message)
```

| Параметр | Тип | Описание |
| --- | --- | --- |
| message | String | Сообщение. |

### Смотрите также

* class [SanitizationException](../../sanitizationexception)
* пространство имен [Aspose.Pdf.Sanitization](../../sanitizationexception)
* сборка [Aspose.PDF](../../../)

---

## SanitizationException(string, Exception) {#constructor_3}

Инициализирует новый экземпляр[`SanitizationException`](../../sanitizationexception) класс с указанным сообщением об ошибке и ссылкой на внутреннее исключение, которое является причиной этого исключения.

```csharp
public SanitizationException(string message, Exception innerException)
```

| Параметр | Тип | Описание |
| --- | --- | --- |
| message | String | Сообщение об ошибке, объясняющее причину исключения. |
| innerException | Exception | Исключение, являющееся причиной текущего исключения, или пустая ссылка (Nothing в Visual Basic), если внутреннее исключение не указано. |

### Смотрите также

* class [SanitizationException](../../sanitizationexception)
* пространство имен [Aspose.Pdf.Sanitization](../../sanitizationexception)
* сборка [Aspose.PDF](../../../)

---

## SanitizationException(Exception) {#constructor_1}

Инициализирует новый экземпляр[`SanitizationException`](../../sanitizationexception) класс с указанным сообщением об ошибке и ссылкой на внутреннее исключение, которое является причиной этого исключения.

```csharp
public SanitizationException(Exception innerException)
```

| Параметр | Тип | Описание |
| --- | --- | --- |
| innerException | Exception | Исключение, являющееся причиной текущего исключения, или пустая ссылка (Nothing в Visual Basic), если внутреннее исключение не указано. |

### Смотрите также

* class [SanitizationException](../../sanitizationexception)
* пространство имен [Aspose.Pdf.Sanitization](../../sanitizationexception)
* сборка [Aspose.PDF](../../../)

<!-- DO NOT EDIT: generated by xmldocmd for Aspose.PDF.dll -->
